CONTRIBUTING.rst

```diff
@@ -54,38 +54,18 @@ Have questions or need help?
 Development Environment
 =======================
 
-You will need Python 2.7 (and optionally a recent version of PyPy) installed on your development machine.
-To run the complete test suite you will also need `ZFS`_ and `Docker`_ installed.
+You will need Python 2.7 installed on your development machine.
+To run the complete test suite you will also need `Docker`_ installed.
 
-The recommended way to get an environment with these installed is to use Vagrant to run a pre-configured Flocker development virtual machine.
-
-First, clone the Flocker repository on your local machine:
-
-.. prompt:: bash $
-
-   git clone https://github.com/ClusterHQ/flocker.git
-   cd flocker
-
-Vagrant 1.6.2 or later is required.
-Once you have Vagrant installed (see the `Vagrant documentation <https://docs.vagrantup.com/v2/>`_) you can run the following to get going:
-
-.. prompt:: bash $
-
-   vagrant up
-   vagrant ssh
-
-The ``flocker`` directory created above will be shared in the virtual machine at ``/vagrant``.
 Install Flocker's development dependencies in a ``virtualenv`` by running the following commands:
 
 .. prompt:: bash $
 
-   cd /vagrant
```

Comment:
Well, the dev-box hasn't changed for quite some time, and I'm not proposing that we delete it. Also, now that we have started to use packer elsewhere, it might make more sense to use that to build a vagrant box for development, instead?